### ISSUE TITLE
Focus Style - Accessibility

### ISSUE THREAD

**Issue by wilsonrocks** [luukdv/hamburger-react#80]
Apologies if this is already addressed, but I couldn't see it.

I saw in Firefox accessibility checker that the button is focusable but may be missing focus styles, linking to https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard?utm_source=devtools&utm_medium=a11y-panel-checks-keyboard#focusable_element_must_have_focus_styling

This does appear to be true - if I tab across to it, it doesn't look visibly different.

```
{
    "name": "menu",
    "role": "button",
    "actions": [
        "Click"
    ],
    "value": "",
    "nodeCssSelector": ".hamburger-react",
    "nodeType": 1,
    "description": "",
    "keyboardShortcut": "",
    "childCount": 0,
    "indexInParent": 0,
    "states": [
        "collapsed",
        "focusable",
        "opaque",
        "enabled",
        "sensitive",
        "expandable"
    ],
    "children": [],
    "attributes": {
        "margin-left": "0px",
        "text-align": "right",
        "text-indent": "0px",
        "formatting": "block",
        "tag": "div",
        "class": "hamburger-react",
        "xml-roles": "button",
        "margin-top": "0px",
        "margin-bottom": "0px",
        "margin-right": "0px",
        "display": "block",
        "explicit-name": "true"
    }
}```
